packages/ui-components-e2e/src/integration/snackbar.spec.ts

```diff
@@ -1,17 +1,17 @@
-describe('buttons', () => {
+describe('snackbar', () => {
   beforeEach(() => {
     cy.visit('/iframe.html?id=components-snackbars--snackbars&args=&viewMode=story');
   });
 
   it('should render a snackbar and then should disappear after 5 seconds', () => {
-    cy.contains('Default').click();
+    cy.get('.lc-button').contains('Default').click();
```

Comment:
This test was failing because it couldn't find the element when we just target `cy.contains('Default')`, but targeting `.lc-button` fixes it. I went ahead and applied this to the rest of the tests in this file.